### ISSUE TITLE
import gmsh dumps core

### ISSUE THREAD

**Issue by gdmcbain** [kinnala/scikit-fem#187]
While experimenting with the Gmsh Python API #180, core was consistently dumped on the `import` statement, depending on the order of importation, but with an inconsistent error message. 

Importing `gmsh` after `skfem` dumps core.

```
>>> import skfem
>>> import gmsh
Erreur de segmentation (core dumped)
```

The other error message encountered is
```
>>> import skfem
>>> import gmsh
free(): invalid pointer
Abandon (core dumped)
```

Importing `gmsh` after `skfem` doesn't dump core, unless `scipy.sparse` has been imported.
```
>>> import scipy.sparse
>>> import gmsh
>>> import skfem
Erreur de segmentation (core dumped)
```

I suspect that it's gmsh that's at fault.  I'll report it to the [gmsh](http://onelab.info/mailman/listinfo/gmsh) mailing list.

**Comment by gdmcbain** [comment]
The above were all on Ubuntu 19.04 with Python 3.7.3 from Miniconda.  Repeating on Microsoft Windows 10 with Python 3.6.6 from Anaconda for Windows, I have not been able to reproduce the error.

**Comment by gdmcbain** [comment]
I close this, assuming it's unrelated to scikit-fem, though it will hold up #180.

**Comment by gdmcbain** [comment]
This might be related to a similar sounding issue 

> when I try to import both gmsh and matplotlib into the same script. Firstly, if matplotlib is imported before gmsh, a Segmentation fault error occurs

reported on the Gmsh mailing list [2019-10-25](http://onelab.info/pipermail/gmsh/2019/013353.html).

The response there was that this may be fixed in recent Gmsh 4.5 snapshots so it might be reasonable to resume work on using the Gmsh Python API #180 here.
